securesystemslib/formats.py

```diff
@@ -193,7 +193,7 @@
 # Supported securesystemslib key types.
 KEYTYPE_SCHEMA = SCHEMA.OneOf(
   [SCHEMA.String('rsa'), SCHEMA.String('ed25519'), SCHEMA.String('ecdsa'),
-   SCHEMA.RegularExpression(r'ecdsa-sha2-nistp(256|384)')])
+   SCHEMA.RegularExpression(r'ecdsa-sha2-nistp(256|384)'), SCHEMA.String('sphincs-shake-128s')])
```

Comment:
Does it make sense to call the keytype just `sphincs`? (see discussion about keytype vs. scheme in #239 and #308)